src/javascript/app_2/Stores/Modules/Portfolio/portfolio_store.js

```diff
@@ -62,6 +63,7 @@ export default class PortfolioStore extends BaseStore {
         } else if (act === 'sell') {
             this.removePositionById(contract_id);
         }
+        GTM.pushTransactionData(response, { bom_ui: 'new' });
```

Comment:
Would be more reliable if calling this at a higher level like [`socket_general`](https://github.com/binary-com/binary-static/blob/master/src/javascript/app_2/Services/socket_general.js).